__fixtures__/includes/nested/service.conf

```diff
@@ -0,0 +1,13 @@
+[SERVICE]
+    flush           1
+    log_level       Debug
+    # Default parser from Fluent Bit
+    parsers_file    ../parsers.conf
```

Comment:
So the include is relative to this file rather than the calling context?
If we include this file from the directory above this path here would now be wrong so it must be relative to this specific file.

Reply:
https://github.com/calyptia/fluent-bit-config-parser/issues/18

Reply:
https://github.com/calyptia/fluent-bit-config-parser/issues/19